PRODUCTIONISATION.md

```diff
@@ -19,26 +19,26 @@
 
 Ensuring that your project adheres to the following RFCs is a good first step as these will be referenced further below:
 
+* [RFC 008 - Server naming convention][rfc_008]
 * [RFC 011 - Editorconfig][rfc_011]
 * [RFC 012 - Makefile standards][rfc_012]
 * [RFC 013 - Code repository standards][rfc_013]
 
 ### 1. General requirements
 
 * [ ] 1.1 Application source code hosted with a Git VCS service
-* 1.2 No linting violations
```

Comment:
I'm curious for the justification for removing linting entirely? It's a smell for a project I think if this best practice isn't followed and there are no coding standards that are adhered to.

Reply:
The reason for removing this is to make the checklist more language agnostic. For example, C# does not require it. There is a section at the bottom linking to the language where linting might be considered such as the Ruby reference we have now.

Reply:
From a coding style perspective, linting can be helpful for C# projects to ensure consistency within a development team.

Reply:
Can you go into the details as to why linting/style guidelines/best practices doesn't apply to c#?

Aware we've never provided references for it before as c# is heavily IDE based, but I don't think that calls for removal. 

It might be best to be a bit looser on the wording instead, and go for "provides a style guideline" and "do's and don't" (which amounts to linting in most situations)

Reply:
I agree that we need a style guideline

Reply:
See https://github.com/madetech/productionisation/pull/51/commits/25720ea6b439d9ec94512ab3ff2cd7b0819d5f9b for changes.